packages.md

```diff
@@ -8,8 +8,7 @@
 | [`Azure.Core`]( sdk/core/Azure.Core/src/Azure.Core.csproj )| [Readme](sdk/core/Azure.Core/README.md) |  N/A  | [Nuget]( https://www.nuget.org/packages/Azure.Core/ ) |
 | [`Azure.Core.Extensions`]( sdk/core/Azure.Core.Extensions/src/Azure.Core.Extensions.csproj )|  N/A  |  N/A  | [Nuget]( https://www.nuget.org/packages/Azure.Core.Extensions/ ) |
 | [`Azure.Identity`]( sdk/identity/Azure.Identity/src/Azure.Identity.csproj )| [Readme](sdk/identity/Azure.Identity/README.md) |  N/A  | [Nuget]( https://www.nuget.org/packages/Azure.Identity/ ) |
-| [`Azure.Messaging.EventHubs`]( sdk/eventhub/Azure.Messaging.EventHubs/src/Azure.Messaging.EventHubs.csproj )| [Readme](sdk/eventhub/Azure.Messaging.EventHubs/README.md) |  N/A  | [Nuget]( https://www.nuget.org/packages/Azure.Messaging.EventHubs/ ) |
-| [`Azure.Messaging.EventHubs.TrackOne`]( sdk/eventhub/Azure.Messaging.EventHubs/src/TrackOneClient/Azure.Messaging.EventHubs.TrackOne.csproj )| [Readme](sdk/eventhub/Azure.Messaging.EventHubs/README.md) |  N/A  |  N/A  |
```

Comment:
This was never intended as an actual package; it was the result of a mistake during the first preview release.